test/symbols.test.js

```diff
@@ -10,8 +10,11 @@ describe('symbols', () => {
   it('should return error', (done) => {
     const compiler = pack(
       'symbols',
-      {},
-      { context: join(__dirname, 'fixtures/[symbols]') }
```

Comment:
Keep this test, just add new

Reply:
This test is the point of the issue, I guess.
I think so because this one was added after #93 as check for problem